sprint01/task06/index.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/style.css">
+    <title>Oscorp</title>
+</head>
+<body>
+    <header class="head">
+        <p>OSCORP</p>
+        <a class="link1" href="#">About</a>
+        <a href="#">Service</a>
+        <a href="#">Contact</a>
+        <a class="link2" href="#">Search</a>
+        <p>Phone:+777777777</p>
+    </header>
+    <nav>
+     <ul class="breadcrumb">
+        <li><a href="#">Home</a></li>
+        <li><a href="#">Catalog</a></li>
+      </ul>
+    </nav>
+       <div class="filtr">
```

Comment:
what is `filtr`?
probably you meant `filter`
fix it please and don't forget about style-naming in `.css`